<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WELCOME</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #424242;
background-color: rgb(0, 0, 0);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Login_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(253, 58, 115);
font: 75 60pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>550</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(253, 58, 115);
	border-radius: 30px;
	font: 14pt &quot;Arial&quot;;
}
QPushButton:hover{
	background-color: rgb(253, 141, 174);
}
</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Username_LineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>270</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background: #424242;
	border: 2px solid rgb(253, 141, 174);
	border-radius: 20px;
	padding-left: 10px;
	font: 14pt &quot;Arial&quot;;
	color: rgb(255, 255, 255);
}
QLineEdit:focus{
	border: 2px solid #FD3A73;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password_LineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background: #424242;
	border: 2px solid rgb(253, 141, 174);
	border-radius: 20px;
	padding-left: 10px;
	font: 14pt &quot;Arial&quot;;
	color: rgb(255, 255, 255);
}
QLineEdit:focus{
	border: 2px solid #FD3A73;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="login_error_label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>490</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 65, 32);
font: 13pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="myimqges.qrc"/>
 </resources>
 <connections/>
</ui>
